bundler.d/ovirt.rb

```diff
@@ -1,3 +1,3 @@
 group :ovirt do
-  gem 'rbovirt', '~> 0.0.24'
+  gem 'rbovirt', '~> 0.0.31'
```

Comment:
:+1: - Foreman repos [support up to 0.0.35](https://github.com/theforeman/foreman-packaging/tree/rpm/develop/rubygem-rbovirt) (just a sanity check so I remember this on the next review)